Segmentation violation when no services in the cluster.

# What occurs

When no services in the cluster, get service command fails with segmentation violation.

```
$ ecsher get svc
Cluster: test
0xc000010188
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x18 pc=0xa4e9da]

goroutine 1 [running]:
github.com/Mic-U/ecsher/ecs.DescribeService(0x0, 0x0, 0xc000037c70, 0x4, 0xff1be0, 0x0, 0x0, 0x0, 0x0, 0x0, ...)
        /workspaces/ecsher/ecs/service.go:38 +0x13a
github.com/Mic-U/ecsher/ecs.GetService(0x0, 0x0, 0xc000037c70, 0x4, 0xff1be0, 0x0, 0x0, 0xe, 0x0, 0x0, ...)
        /workspaces/ecsher/ecs/service.go:24 +0x250
github.com/Mic-U/ecsher/cmd.getService()
        /workspaces/ecsher/cmd/get.go:97 +0x15c
github.com/Mic-U/ecsher/cmd.glob..func3(0xfb71a0, 0xc00006d640, 0x1, 0x1)
        /workspaces/ecsher/cmd/get.go:29 +0xb1
github.com/spf13/cobra.(*Command).execute(0xfb71a0, 0xc00006d610, 0x1, 0x1, 0xfb71a0, 0xc00006d610)
        /go/pkg/mod/github.com/spf13/cobra@v1.1.3/command.go:856 +0x2c2
github.com/spf13/cobra.(*Command).ExecuteC(0xfb7420, 0x0, 0x0, 0x0)
        /go/pkg/mod/github.com/spf13/cobra@v1.1.3/command.go:960 +0x375
github.com/spf13/cobra.(*Command).Execute(...)
        /go/pkg/mod/github.com/spf13/cobra@v1.1.3/command.go:897
github.com/Mic-U/ecsher/cmd.Execute()
        /workspaces/ecsher/cmd/root.go:30 +0x2d
main.main()
        /workspaces/ecsher/main.go:21 +0x25
exit status 2
```

# Expected behavior

It is expected that ecsher says 'No services found' and exit.

```
$ ecsher get svc
Cluster: test
No services found
```

--- a/ecs/cluster.go
+++ b/ecs/cluster.go
@@ -15,6 +15,9 @@ func GetCluster(region string, names []string) ([]ecsTypes.Cluster, error) {
 		if err != nil {
 			return nil, err
 		}
+		if len(listClustersOutput.ClusterArns) == 0 {
+			return []ecsTypes.Cluster{}, nil
+		}
 		return DescribeCluster(region, listClustersOutput.ClusterArns)
 	}
 	return DescribeCluster(region, names)

--- a/ecs/service.go
+++ b/ecs/service.go
@@ -19,6 +19,9 @@ func GetService(region string, cluster string, names []string) ([]ecsTypes.Servi
 		if err != nil {
 			return nil, err
 		}
+		if len(listServicesOutput.ServiceArns) == 0 {
+			return []ecsTypes.Service{}, nil
+		}
 		return DescribeService(region, cluster, listServicesOutput.ServiceArns)
 	}
 	return DescribeService(region, cluster, names)

--- a/ecs/task.go
+++ b/ecs/task.go
@@ -29,8 +29,7 @@ func GetTaskInCluster(region string, cluster string, names []string) ([]ecsTypes
 			return nil, err
 		}
 		if len(listTasksOutput.TaskArns) == 0 {
-			result := []ecsTypes.Task{}
-			return result, nil
+			return []ecsTypes.Task{}, nil
 		}
 		return DescribeTask(region, cluster, listTasksOutput.TaskArns)
 	}

